docs/sections/pipeline_samples/tutorials/generate_textcat_dataset.ipynb

```diff
@@ -0,0 +1,1005 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.            "task": "Determining the topic of a news article as sports, technology, or business"
</pre><p>I think you forgot to include "world" here. Also, perhaps we can generalize it  a bit by directly using the labels variable? Why do we use determine vs classify vs categorizig etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/949/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>